lib/erb_lint/cli.rb

```diff
@@ -95,13 +112,47 @@ def run(args = ARGV)
 
     private
 
+    attr_reader :cache, :config
+
+    def run_on_file(runner, filename)
+      file_content = read_content(filename)
+      if with_cache? && !autocorrect?
+        run_using_cache(runner, filename, file_content)
+      else
+        file_content = run_with_corrections(runner, filename, file_content)
+      end
+      log_offense_stats(runner, filename)
+      file_content
+    end
+
+    def run_using_cache(runner, filename, file_content)
+      if cache.include?(filename) && !autocorrect?
```

Comment:
This method is never called with `autocorrect?` so we don't need to check again.
```suggestion
      if cache.include?(filename)
```

Reply:
Good catch! I removed it in the latest update.